src/main/groovy/org/mockito/release/notes/improvements/GitHubTicketFetcher.java

```diff
@@ -32,24 +34,26 @@
                     .browse();
 
             while (!tickets.isEmpty() && issues.hasNextPage()) {
-                List<JSONObject> page = issues.nextPage();
+                List<JsonObject> page = issues.nextPage();
 
                 out.addAll(extractImprovements(
                         dropTicketsAboveMaxInPage(tickets, page),
                         page, onlyPullRequests));
             }
         } catch (Exception e) {
-            throw new RuntimeException("Problems fetching " + ticketIds.size() + " from GitHub", e);
+            String message = "Problems fetching " + ticketIds.size() + " tickets from GitHub";
+            LOG.info(message, e);
```

Comment:
Given that we throw an exception, let's avoid also logging the message. Otherwise we might have confusing and untidy dual stack trace when it fails. "Log XOR throw" :)